src/Access/AccessControl.cpp

```diff
@@ -105,6 +110,8 @@ class AccessControl::CustomSettingsPrefixes
     {
         if (Settings::hasBuiltin(setting_name))
             return true;
+        if (setting_name.starts_with(MERGE_TREE_SETTINGS_PREFIX) && MergeTreeSettings::hasBuiltin(setting_name.substr(strlen(MERGE_TREE_SETTINGS_PREFIX))))
```

Comment:
if you have `MERGE_TREE_SETTINGS_PREFIX` as `constexpr std::string_view` you can use `size` instead of `strlen`.

Reply:
Changed to `string_view`. But in another place it is now ugly, due to operator+ having no overload for `string_view` and `string`